src/main.js

```diff
@@ -250,14 +252,32 @@ $(document).ready(() => {
            */
           isMouseInSidebar = true;
           clearTimer();
-          if (!isSidebarVisible()) toggleSidebar(true);
```

Comment:
why remove this?

Reply:
It causes unexpectedly opening the sidebar when hovering around the edge of the browser ( outside the toggler ).

Reply:
But what's the effect of not having it? Why did we have it in the first place?

Reply:
![Screen Shot 2019-04-14 at 4 43 14 PM](https://user-images.githubusercontent.com/8633840/56091083-6b6e9e00-5ed4-11e9-87ed-0e28278aea6c.png)

There 2 explanations. 
 1. It has been there all the time and we've missed the edge case.
 2. The resizable border gets exposed ( check the screenshot ) and it causes the effect.

Reply:
I'm afraid removing this will cause a minor annoyance. Currently, when the tree starts hiding, if I hover, it will expand again. But with this change, that won't happen correct? The tree will just collapse complete, then users would have to either hover or click the toggler.